### PR TITLE
Webview Tests

### PR DESCRIPTION
**Pull request by ecmel**
### Changes introduced by this PR

- Fix for KXI-33307
- Foundation for further webview tests

### CODE REVIEW

File: test/suite/webview.test.ts
Comment (Philip-Carneiro-KX) [inline test/suite/webview.test.ts:1]: image
Like I said in the ticket comment

File: src/webview/components/kdbDataSourceView.ts [hunk 1]
```diff
@@ -214,24 +215,22 @@ export class KdbDataSourceView extends LitElement {
...
-                : api.api;
-
-            return html`
-              <vscode-option value="${value}" ?selected="${value === selected}"
-                >${value}</vscode-option
-              >
-            `;
-          })}
-      `;
+      return this.insightsMeta.api
```
Comment (Philip-Carneiro-KX) [inline src/webview/components/kdbDataSourceView.ts:218]: why this change? if we remove it the html response extensions that make more easy/readable to maintain it later

**Review by Philip-Carneiro-KX — CHANGES_REQUESTED** [review]

File: test/suite/webview.test.ts
Comment (ecmel) [reply test/suite/webview.test.ts:1]: These tests uses mocking for some esm modules so it is better to keep these tests separate for the moment.

File: src/webview/components/kdbDataSourceView.ts (hunk 1 above)
Comment (ecmel) [reply src/webview/components/kdbDataSourceView.ts:218]: Outer html``is superfluous as you may check from the tests.

**Review by Philip-Carneiro-KX — APPROVED** [review]

**Pull request merged**